src/lib.rs

```diff
@@ -112,39 +112,84 @@ fn quad(w: f32, h: f32, pivot: Option<Vec2>, double_sided: bool) -> Mesh {
     let h2 = h / 2.0;
     let mut mesh = Mesh::new(PrimitiveTopology::TriangleList);
     let vertices = match pivot {
-        None => { vec![[-w2, -h2, 0.0], [w2, -h2, 0.0], [-w2, h2, 0.0], [w2, h2, 0.0],
-                       [-w2, -h2, 0.0], [w2, -h2, 0.0], [-w2, h2, 0.0], [w2, h2, 0.0]] },
+        None => {
```

Comment:
There's a few changes (like this) that don't actually change the code, just spread it over more vertical area. Revert to keep the PR single-purpose.